code/go/0chain.net/validator/main.go

```diff
@@ -188,8 +188,8 @@ func SetupValidatorOnBC(logDir string) {
 	var logName = logDir + "/validator.log"
 	zcncore.SetLogFile(logName, false)
 	zcncore.SetLogLevel(3)
-	zcncore.InitZCNSDK(serverChain.BlockWorker, config.Configuration.SignatureScheme)
-	zcncore.SetWalletInfo(node.Self.GetWalletString(), false)
+	zcncore.InitZCNSDK(serverChain.BlockWorker, config.Configuration.SignatureScheme) //nolint:errcheck // error would be detected later as SDK not initialized
```

Comment:
Pass these error up to main and error out from there. The user will likley find this error more useful than any follow up error. Same for wallet infol